lapis-e2e/test/nucleotideInsertions.spec.ts

```diff
@@ -54,23 +54,25 @@ describe('The /nucleotideInsertions endpoint', () => {
     const resultWithLimit = await lapisClient.postNucleotideInsertions({
       insertionsRequest: {
         orderBy: [{ field: 'count', type: 'ascending' }],
-        limit: 2,
+        limit: 4,
       },
     });
 
-    expect(resultWithLimit.data).to.have.length(2);
-    expect(resultWithLimit.data[1]).to.have.property('insertion', 'ins_22339:GCTGGT');
+    expect(resultWithLimit.data).to.have.length(4);
+    expect(resultWithLimit.data[0]).to.have.property('count', 1);
+    expect(resultWithLimit.data[3]).to.have.property('count', 17);
+    expect(resultWithLimit.data[3]).to.have.property('insertion', 'ins_25701:CCC');
 
     const resultWithLimitAndOffset = await lapisClient.postNucleotideInsertions({
       insertionsRequest: {
-        orderBy: [{ field: 'count', type: 'ascending' }],
-        limit: 2,
+        orderBy: [{ field: 'count', type: 'descending' }],
+        limit: 4,
         offset: 1,
       },
     });
```

Comment:
This request is supposed to be the same as the first in this test. IMO reverting order makes it too confusing.

Did you also change it due to the "the first two positions both have count 1 so their order could change" thing? I think the better alternative would be to further specify the order to make it non-ambiguous (e.g. additionally ordering by insertion or similar).